language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/AssignmentStatementNodeContext.java

```diff
@@ -67,7 +68,8 @@ public AssignmentStatementNodeContext() {
             (2) [module:]TypeName c = module:a<cursor>
              */
             QualifiedNameReferenceNode qNameRef = (QualifiedNameReferenceNode) node.expression();
-            Predicate<Symbol> filter = symbol -> symbol.kind() == SymbolKind.VARIABLE;
+            Predicate<Symbol> filter = symbol -> symbol instanceof VariableSymbol
```

Comment:
Can't we use `SymbolKind.VARIABLE` instead?

Reply:
No, if we use the VARIABLE symbol kind, then it will not capture the constants as per the current model. In order to avoid having two check with constant and variable kinds, added the instance of check

Reply:
If the intention is to address both constants and variables I would still prefer two checks; since `==` comparison is far efficient  compared to `instanceOf` and for the better readability of allowing constants and variables.

Reply:
Yes, agreed. This will be addressed along with the newly proposed model